RoslynPluginGenerator/UIResources.resx

```diff
@@ -260,7 +260,7 @@ The conflict will be ignored.</value>
     <value>Maximum supported Roslyn version:</value>
   </data>
   <data name="Program_SupportedSonarQubeVersions" xml:space="preserve">
-    <value>Minimum supported SonarQube version: 7.9.6 (highest version tested against: 9.1)</value>
+    <value>Minimum supported SonarQube version: 7.9.6 (highest version tested against: 9.7)</value>
```

Comment:
SonarQube v9.7 was released today, so I tested against it and everything still works as expected.

Reply:
This message is displayed in the console on startup e.g. 
![image](https://user-images.githubusercontent.com/10757559/196503265-542c829f-555d-4451-9661-9ae82ea54b02.png)

(FYI the version numbers for the plugin generator and support Roslyn version are extract programmatically from the relevant assemblies)